contracts/migrations/7_create_token_whitelist.js

```diff
@@ -5,23 +5,12 @@ const V00_Marketplace = artifacts.require('./V00_Marketplace.sol')
 const whitelistExpiration = Date.parse('28 Feb 2019 00:00:00 PST') / 1000
 
 module.exports = function(deployer, network) {
-  return deployer.then(() => {
-    switch(network) {
-      case "rinkeby":
-      case "ropsten":
-        return createTokenWhitelistForTestnets(network)
-
-      case "mainnet":
-        console.error('\n\n*** WHITELIST MIGRATION NOT YET IMPLEMENTED FOR MAINNET ***\n\n')
-        return
-
-      default:
-        console.log(`Skipping whitelist creation for network ${network}`)
-    }
-  })
+  if (network !== 'development' || process.env['SIMULATE_MAINNET']) {
+    return createTokenWhitelist(network)
+  }
 }
 
-async function createTokenWhitelistForTestnets() {
+async function createTokenWhitelist(network) {
   const token = await OriginToken.deployed()
   const tokenOwner = await token.owner()
 
```

Comment:
We won't have a faucet on Mainnet. So should this be only executed if network is not mainnet ?

Reply:
`transferTokensAndContractsToMultiSig()` in migration 9 removes the token deployer from the whitelist for mainnet, so I think this is fine

Reply:
the deployer also needs to transfer all its OGN to the multi-sig wallet, so it needs to be whitelisted temporarily